near-token-factory/src/lib.rs

```diff
@@ -124,6 +124,7 @@ impl Contract {
             // batched transaction.
             Promise::new(token_account_id)
                 .create_account()
+                .transfer(3_000_000_000_000_000_000_000_000) // TODO: cost for storage should be covered by user?
```

Comment:
The new token contract needs NEAR to cover its storage staking. I added this line to make the test pass, but it's maybe better if the user who creates the token covers the cost instead of the Factory contract. This is another reason to separate create from deposit.